AspNetCore.sln

```diff
@@ -1553,6 +1572,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Mvc.Te
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Mvc.Testing.Tasks", "Mvc.Testing.Tasks", "{34465A80-3BAF-4059-81B3-9090C0DB598F}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestTasks", "src\Servers\IIS\IIS\test\testassets\TestTasks\TestTasks.csproj", "{9B8F871E-ED33-4D2F-AA49-E39D9299EC85}"
```

Comment:
@dougbu IIRC you removed this from the solution for some specific reason. Do you happen to remember?

Reply:
I don't think this project has been in the solution since @Pilchie consolidated our solutions. I also don't see any history of me changing the old src/Servers/IIS/IISIntegration.sln. Could you be more specific to help me remember❔

Reply:
Nvrm, I think it was https://github.com/dotnet/aspnetcore/pull/27597/files. We're good.